index.html

```diff
@@ -321,6 +321,88 @@ <h2>Securing JSON-LD VCs with COSE</h2>
       </section>
     </section>
 
+
+
+<section class="normative">
+  <h2>Key Discovery</h2>
+  <p>
+In order to complete the <a data-cite="VC-DATA-MODEL#dfn-verify">verification</a> process, 
+a <a data-cite="VC-DATA-MODEL#dfn-verifier">verifier</a> needs to obtain the cryptographic keys used to secure the 
+<a data-cite="VC-DATA-MODEL#dfn-credential">credential</a>.
+  </p>
+  <p>
+There are several different ways to discover the verification keys of
+the <a data-cite="VC-DATA-MODEL#dfn-issuers">issuers</a>
+and <a data-cite="VC-DATA-MODEL#dfn-holders">holders</a>.
+  </p>
+
+  <section>
+    <h2>Registered Claim Names</h2>
+    <p>
+When found in the <a data-cite="RFC7515#section-4.1">Protected Header</a>, or 
+the <a data-cite="RFC7519#section-4.1.1">Protected Claimset</a>, members present in
+<a href="https://www.iana.org/assignments/jwt/jwt.xhtml">IANA Assignments for JSON Web Token (JWT)</a> and 
+<a href="https://www.iana.org/assignments/jose/jose.xhtml">IANA Assignments for JSON Object Signing and Encryption (JOSE)</a>
+are to be interpreted according to the associated specifications referenced by IANA.
+    </p>
+    <p>
+<a href="#registered-claim-names">Registered claims</a> that are present in either
+the <a data-cite="RFC7515#section-4.1">Protected Header</a>
+or the <a data-cite="RFC7519#section-4.1.1">Claimset</a> can be used to help
+<a data-cite="VC-DATA-MODEL#dfn-verifier">verifiers</a> discover verification keys.
+    </p>
+    <section>
+      <h2>kid</h2>
+      <p>
+If <code>kid</code> is present in the <a data-cite="RFC7515#section-4.1">Protected Header</a>,
+a <a data-cite="VC-DATA-MODEL#dfn-verifier">verifier</a> can use this parameter
+to obtain a <a data-cite="RFC7517#section-4">JSON Web Key</a> to use in the 
+<a data-cite="VC-DATA-MODEL#dfn-verify">verification</a> process.
+      </p>
+    </section>
+    <section>
+      <h2>iss</h2>
+      <p>
+If <code>iss</code> is present in the <a data-cite="RFC7515#section-4.1">Protected Header</a>
+or the <a data-cite="RFC7519#section-4.1.1">JWT Claims </a>,
+a <a data-cite="VC-DATA-MODEL#dfn-verifier">verifier</a> can use this parameter
+to obtain a <a data-cite="RFC7517#section-4">JSON Web Key</a> to use in the 
+<a data-cite="VC-DATA-MODEL#dfn-verify">verification</a> process.
```

Comment:
```suggestion
If <code>iss</code> is present in the <a data-cite="RFC7515#section-4.1">Protected Header</a>
a <a data-cite="VC-DATA-MODEL#dfn-verifier">verifier</a> can use this parameter
to obtain a <a data-cite="RFC7517#section-4">JSON Web Key</a> to use in the 
<a data-cite="VC-DATA-MODEL#dfn-verify">verification</a> process. To avoid confusion
or inconsistency with an <code>iss</code> value in the
<a data-cite="RFC7515#section-4.1">Protected Header</a> or with the issuer
identifier expressed in the Verifiable Credential itself, <code>iss</code> MUST NOT be present
in the <a data-cite="RFC7519#section-4.1.1">JWT Claims</a>.
```

Reply:
I would prefer to say "in the Verifiable Credential" instead of "JWT Claims" for reasons previously mentioned in other issues, but if using "JWT Claims" is my only option here to address the issue, it's the best option.

Reply:
I won't apply this change because it assumes specific processing which is not required.

It's true that `iss` can be present in both the claimset and the header.

When its present in either, it needs to be interpreted according to the registry or security faults will be introduced.

It is true that we *like* the idea of not putting this in the claimset, but that is a recommendation that I am not trying to make in this PR...

I am trying to document existing behavior first, and then make recommendations second.

@dlongley would an issue marker like this work for you instead of accepting your change suggestion:

```
<pre class="issue">
The working group is discussing if we should recommend 
against using registered claim names in the claimset, 
and recommend they only be present in the protected header.
</pre>
```

Hopefully it captures what you are trying to recommend.

Reply:
@dlongley perhaps this context helps explain my rational for declining your suggestion...

https://github.com/w3c/vc-jwt/pull/111#discussion_r1248288752

I suspect you might want to recommend against some of the scenarios where `iss` in a `claimset` is used to discover keys for an issuer... there will be future PRs that make it much easier to tackle that issue by itself, if that is your concern.

Reply:
@OR13,

I'm happy for you to do an issue marker or if you think that people really won't be confused here and we don't need any advice, I'll follow your lead as well. I'm just trying to raise possible problem points to ensure we improve upon what happened in 1.1. I'll approve this PR now and let you decide what to do from there. Thanks!

Reply:
Ok, I think we don't need an issue marker, but I will try to draw your attention to where this comes up in future PRs (it will come up again)... perhaps we can find better text with additional context around the term in those PRs.